src/trio/testing/_raises_group.py

```diff
@@ -280,8 +380,6 @@ class RaisesGroup(Generic[BaseExcT_co]):
 
     It currently does not care about the order of the exceptions, so ``RaisesGroups(ValueError, TypeError)`` is equivalent to ``RaisesGroups(TypeError, ValueError)``.
```

Comment:
I think we're leaning more into not caring. Should this be updated:

```suggestion
    It does not care about the order of the exceptions, so ``RaisesGroups(ValueError, TypeError)`` is equivalent to ``RaisesGroups(TypeError, ValueError)``.
```